desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -117,16 +164,26 @@ void TabDiveInformation::updateData()
 	updateWhen();
 	ui->waterTemperatureText->setText(get_temperature_string(current_dive->watertemp, true));
 	ui->airTemperatureText->setText(get_temperature_string(current_dive->airtemp, true));
-
 	if (current_dive->salinity)
 		ui->salinityText->setText(QString("%1g/ℓ").arg(current_dive->salinity / 10.0));
 	else
 		ui->salinityText->clear();
-
-	ui->atmPressType->setEditable(true);
 	ui->atmPressType->setItemText(1, get_depth_unit());  // Check for changes in depth unit (imperial/metric)
-	ui->atmPressType->setEditable(false);
 	ui->atmPressType->setCurrentIndex(0);  // Set the atmospheric pressure combo box to mbar
+	if (current_dive->salinity) {			// Set water type indicator (EN13319 = 1.020 g/l)
+		if (current_dive->salinity < 10050) 
+			ui->waterTypeText->setText("Fresh");
+		else if (current_dive->salinity < 10190)
+			ui->waterTypeText->setText("Salty");
```

Comment:
`Salty` ?? :-)
Also, those texts need to be translated, so `tr("Fresh")`. etc.

Reply:
Done. I have kept salty by lack of a good alternative "salty-ish" or "Brack" do not sound good.